internal/npm_install/generate_build_file.js

```diff
@@ -108,7 +113,8 @@ function generateRootBuildFile(pkgs) {
 # this filegroup to include those files.
 filegroup(
     name = "node_modules",
-    srcs = glob(["node_modules/.bin/*"]) + [
+    srcs = [
+        ${binFiles}
```

Comment:
nit: to be consistent, should format this one in this presentation logic too (move the join on newline/spaces)

Reply:
👍